Failing test: APM Cypress Tests.x-pack/plugins/observability_solution/apm/ftr_e2e/cypress/e2e/dependencies/dependencies·cy·ts- Dependencies with high volume of data shows dependencies inventory page

A test failed

```
AssertionError: Timed out retrying after 30000ms: Expected to find content: 'Page 1 of 60' within the selector: 'nav' but never did.
--
  | at Context.eval (webpack:///./cypress/e2e/dependencies/dependencies.cy.ts:152:7)

```

Failure: [Build - unsupported ftrs main](https://buildkite.com/elastic/kibana-on-merge-unsupported-ftrs/builds/13904#018e5c15-b1f2-408b-a206-158cdc7f0d87/268-1878)

--- a/x-pack/plugins/observability_solution/apm/ftr_e2e/cypress/e2e/dependencies/dependencies.cy.ts
+++ b/x-pack/plugins/observability_solution/apm/ftr_e2e/cypress/e2e/dependencies/dependencies.cy.ts
@@ -112,8 +112,7 @@ describe('Dependencies', () => {
   });
 });
 
-// FLAKY: https://github.com/elastic/kibana/issues/179083
-describe.skip('Dependencies with high volume of data', () => {
+describe('Dependencies with high volume of data', () => {
   before(() => {
     synthtrace.index(
       generateManyDependencies({

